src/core/hle/service/time/steady_clock_core.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 yuzu emulator team
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "common/uuid.h"
+#include "core/hle/service/time/clock_types.h"
+
+namespace Core {
+class System;
+}
+
+namespace Service::Time::Clock {
+
+class SteadyClockCore {
+public:
+    SteadyClockCore() : clock_source_id{Common::UUID::Generate()}, is_initialized{} {}
```

Comment:
```suggestion
    SteadyClockCore() : clock_source_id{Common::UUID::Generate()} {}
```

This is already initialized in the class definition further down. Technically the initialization of clock_source_id can be moved down there as well, allowing the constructor to be defaulted.